app/routes.js

```diff
@@ -304,6 +304,13 @@ export default config => {
         }}
       </Route>
 
+      <Route
+        path="/FollowTheItinerary"
+        getComponent={() =>
+          import('./component/itinerary/FollowTheItinerary').then(getDefault)
+        }
+      />
```

Comment:
@vesameskanen 's opinion was that we don't need a new route but it's sufficient just to add new state variable to the `ItineraryDetails` component. I have some difficulty to decide what is the most optimal. Having own route for this does have the benefit of the "state" being more permanent in cases where the mobile phone decides to throw the page out of memory and it's loaded with fresh state (however there are currently other potential issues when that happens as it's not guaranteed that the same itinerary will be loaded). Also, users can easily navigate out of the "follow" view by just going back to the previous page if there is a new route.

However, if we use a new route. I think the correct place for it is to be a child route of the `ItineraryDetails` component in this file and the full path would be something like`/${PREFIX_ITINERARY_SUMMARY}/:from/:to/:hash/:secondHash/${ITINERARY_FOLLOW}?`. The `:secondHash` causes some headache and maybe we should try to refactor the routes a bit because of that so it's more clear what is in `hash` or `secondHash`.  We might want to reuse the same map in ItineraryDetails and in the new follow component and not sure what is the optimal way to achieve that.

Reply:
We try to use Finnish path names (although I think we have sometimes failed at it).

Reply:
It would be great, if you could define constants for path names, so forks can replace them by English/local path names.

Reply:
Url path is now at path.js and it's in finnish. 

Currently this hole feature is still under construction by designers. So I guess that many things might change.